core/controllers/profile.py

```diff
@@ -480,6 +485,26 @@ def post(self):
 class SiteLanguageHandler(base.BaseHandler):
     """Changes the preferred system language in the user's preferences."""
 
+    # Currently the language code we use consists
+    # of this 3 types: 1) 'en', 2) 'kab' and 3) 'hi-en'.
+    # Currently we are keeping out language code ISO 639 compatible
+    # We want to impose a regex to check and enforce
+    # the schame types. The regex lives in core/feconf.py.
```

Comment:
This should be in the feconf.py file

Reply:
@vojtechjelinek I also added the comment in core/feconf.py file